src/main/java/org/dependencytrack/model/Finding.java

```diff
@@ -45,181 +46,59 @@
+        optValue(component, "name", findingRow.componentName());
+        optValue(component, "group", findingRow.componentGroup());
+        optValue(component, "version", findingRow.componentVersion());
+        if (findingRow.componentPurl() != null) {
+            optValue(component, "purl", findingRow.componentPurl());
         }
-        optValue(vulnerability, "cwes", cwes);
-        optValue(attribution, "analyzerIdentity", o[25]);
-        optValue(attribution, "attributedOn", o[26]);
-        optValue(attribution, "alternateIdentifier", o[27]);
-        optValue(attribution, "referenceUrl", o[28]);
-
-        optValue(analysis, "state", o[29]);
-        optValue(analysis, "isSuppressed", o[30], false);
-        if (o.length > 33) {
-            optValue(vulnerability, "published", o[31]);
-            optValue(component, "projectName", o[33]);
-            optValue(component, "projectVersion", o[34]);
+        optValue(component, "cpe", findingRow.componentCpe());
+        optValue(component, "project", findingRow.projectUuid());
+
+        optValue(vulnerability, "uuid", findingRow.vulnUuid());
+        optValue(vulnerability, "source", findingRow.vulnSource());
+        optValue(vulnerability, "vulnId", findingRow.vulnId());
+        optValue(vulnerability, "title", findingRow.vulnTitle());
+        optValue(vulnerability, "subtitle", findingRow.vulnSubtitle());
+        optValue(vulnerability, "description", findingRow.vulnDescription());
+        optValue(vulnerability, "recommendation", findingRow.vulnRecommendation());
+        if (findingRow.vulnSeverity() != null) {
+            final Severity severity = findingRow.vulnSeverity();
+            optValue(vulnerability, "severity", severity.name());
+            optValue(vulnerability, "severityRank", severity.ordinal());
         }
+        optValue(vulnerability, "cvssV2BaseScore", findingRow.cvssV2BaseScore());
+        optValue(vulnerability, "cvssV3BaseScore", findingRow.cvssV3BaseScore());
+        optValue(vulnerability, "cvssV2Vector", findingRow.cvssV2Vector());
+        optValue(vulnerability, "cvssV3Vector", findingRow.cvssV3Vector());
+        optValue(vulnerability, "owaspLikelihoodScore", findingRow.owaspRRLikelihoodScore());
+        optValue(vulnerability, "owaspTechnicalImpactScore", findingRow.owaspRRTechnicalImpactScore());
+        optValue(vulnerability, "owaspBusinessImpactScore", findingRow.owaspRRBusinessImpactScore());
+        optValue(vulnerability, "owaspRRVector", findingRow.owaspRRVector());
+        optValue(vulnerability, "epssScore", findingRow.epssScore());
+        optValue(vulnerability, "epssPercentile", findingRow.epssPercentile());
+        optValue(vulnerability, "cwes", findingRow.cwes());
+        addVulnerabilityAliases(findingRow.vulnAliasesJson());
+
+        optValue(attribution, "analyzerIdentity", findingRow.analyzerIdentity());
+        optValue(attribution, "attributedOn", findingRow.attributed_on());
```

Comment:
We will need `Instant` fields to be converted to `Date` here. `Instant`s are JSON-serialized differently, e.g. currently `attributedOn` gets serialized as `1742383948.368253`, when it should be like this: `1742383943589`.

The date now renders wrong for findings: 
![image](https://github.com/user-attachments/assets/bc38a0f2-e982-4218-90a0-ee56b8c1f73d)